index.html

```diff
@@ -30,7 +30,7 @@
 
   <!-- HEADER COMPONENT -->
   <div class="header">
-    <span class="date">SMARCH 28, 2018</span>
+    <span class="date">MARCH 28, 2018</span>
```

Comment:
## bugfix - cleanup
Eustace good work on fixing the look of the date string in this project and going over and re-mediating the code base. you have shown an understanding of the importance of granularity of commits and clear commit messages. keep up the good work